src/applications/disability-benefits/all-claims/config/form.js

```diff
@@ -189,7 +189,8 @@ const formConfig = {
   preSubmitInfo,
   chapters: {
     veteranDetails: {
-      title: isReviewPage => `${isReviewPage ? 'Review ' : ''}Veteran Details`,
+      title: ({ onReviewPage }) =>
```

Comment:
This form.js is the ONLY one using a function that checks whether FormNav's on review-page.  Now we're grabbing onReviewPage out of the props obj.